packages/trader/src/Modules/Trading/Components/Form/Purchase/contract-info.jsx

```diff
@@ -1,24 +1,38 @@
 import classNames from 'classnames';
 import PropTypes from 'prop-types';
 import React from 'react';
-import { Icon, DesktopWrapper, Money, MobileWrapper, Popover, Text } from '@deriv/components';
+import { Icon, DesktopWrapper, Money, MobileWrapper, Popover, Text, Modal } from '@deriv/components';
 import { localize, Localize } from '@deriv/translations';
 import { getCurrencyDisplayCode, getLocalizedBasis } from '@deriv/shared';
 import CancelDealInfo from './cancel-deal-info.jsx';
 
+const PayoutHintModal = ({ is_open, onClose, type }) => (
+    <Modal small is_open={is_open} is_vertical_centered toggleModal={onClose} title={localize('Payout per point')}>
+        <Text size='xxs' as='p' styles={{ padding: '0 1.6rem 1.6rem 1.6rem' }}>
```

Comment:
I think we can create a class for this style. Also, you can omit last `1.6rem`.

Reply:
Ok. Since Text supported style prop, I though of aligning the layout using it